src/greenlight/runner.clj

```diff
@@ -77,6 +81,60 @@
     true))
 
 
+(defmacro ^:private with-delayed-output
+  [printer & body]
+  `(let [out# (java.io.StringWriter.)
+         original-out# *out*
+         original-err# *err*]
+     (binding [*out* out#, *err* out#]
+       (let [result# ~@body]
```

Comment:
This could expand into an invalid let binding since `body` is a varargs sequence. You need:
```suggestion
       (let [result# (do ~@body)]
```